PennMobile/General/UserDefaults + Helpers.swift

```diff
@@ -389,3 +391,47 @@ extension UserDefaults {
         }
     }
 }
+
+// MARK: - Privacy Settings
+extension UserDefaults {
+    func save(_ preferences: PrivacyPreferences) {
+        let encoder = JSONEncoder()
+        if let encoded = try? encoder.encode(preferences) {
+            UserDefaults.standard.set(encoded, forKey: UserDefaultsKeys.privacyPreferences.rawValue)
+        }
+    }
+
+    func getPrivacyPreferences() -> PrivacyPreferences? {
+        let decoder = JSONDecoder()
+        if let decodedData = UserDefaults.standard.data(forKey: UserDefaultsKeys.privacyPreferences.rawValue) {
+            return try? decoder.decode(PrivacyPreferences.self, from: decodedData)
+        }
+        return nil
+    }
+
+    func clearPrivacyPreferences() {
+        removeObject(forKey: UserDefaultsKeys.privacyPreferences.rawValue)
+    }
+}
+
+// MARK: - Notification Settings
+extension UserDefaults {
+    func save(_ preferences: NotificationPreferences) {
+        let encoder = JSONEncoder()
+        if let encoded = try? encoder.encode(preferences) {
+            UserDefaults.standard.set(encoded, forKey: UserDefaultsKeys.notificationPreferences.rawValue)
+        }
```

Comment:
Missing `synchronized()`

Reply:
resolved -- this function no longer needed